docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -16,37 +19,100 @@ For scripts to successfully use BODL to access storefront data, the following mu
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `channel_id` | integer | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | Item on the product page. | See [line_item](#line_item-object-schema) object for event fields |
+
+### View product category
+
+When the shopper clicks a button to view the category page for a product, the following fields appear in the BODL object for the item:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `category_id` | integer | ID of the category. | [Get all categories](/docs/rest-management/catalog/category#get-all-categories) <br /> `data.id` |
+| `category_name` | string | Name of the category. | [Get all categories](/docs/rest-management/catalog/category#get-all-categories) <br /> `data.name` |
+| `channel_id` | integer | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | Items in the category. | See [line_item](#line_item-object-schema) object for event fields |
+
+### Add product to cart
+
+When the shopper clicks a button to add a product to a cart, the following fields appear in the BODL object for the item:
 
 | Event field | Type | Description | BigC data map |
 |:---|:---|:---|:---|
-| `cart_value` | number | Final value of cart after taxes, discounts, and coupons are applied. | [Get a cart](/docs/rest-management/carts/carts-single#get-a-cart) <br /> `cart_amount` |
 | `channel_id` | string | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
-| `coupon_codes[]` | array of strings | Coupon codes applied to the cart. | [Get a cart](/docs/rest-management/carts/carts-single#get-a-cart) <br /> `coupons.code` |
-| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. BigCommerce tracks currency for a cart and not for each line item within the cart. | [Get a cart](/docs/rest-management/carts/carts-single#get-a-cart) <br /> `currency.code` |
+| `currency` | string | [ISO-4217 currency code](https://en.wikipedia.org/wiki/ISO_4217) of the order. | [Get a checkout](/docs/rest-storefront/checkouts#get-a-checkout) <br />  `data.cart.currency.code` |
+| `id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
```

Comment:
Another note. The field is empty when adding a product from PLP and is completely missing if product added from PDP.
@bc-sanjayjoshi

Reply:
@bc-rmalyavc which field is this?

Reply:
@bc-sanjayjoshi This was about the `id` field for the Add Product to Cart event. 

I'll go ahead and resolve the conversation so I can publish the doc, as this is a bug fix (rather than a schema change)